avocado/core/runner.py

```diff
@@ -447,6 +447,7 @@ def sigtstp_handler(signum, frame):     # pylint: disable=W0613
                                            ignore_window)
                         stage_1_msg_displayed = True
                     ignore_time_started = time.time()
+                    os.kill(proc.pid, signal.SIGINT)
```

Comment:
Well we can define any behavior we want and interrupting the test process by SIGTERM looks like sane behavior on ctrl+c sent to the runner. Anyway looking forward to the v2, hopefully it'll cover the hang as well as killing forked processes like this fix.